### DISCUSSION TITLE
Can customer geometry collide with particle system or FEM softbody?

### DISCUSSION THREAD

**Discussion by yananZZZZ** [NVIDIA-Omniverse/PhysX#71] [Q&A]
I create a particle system using functions in the SnippetPBDCloth and add to SnippetCustomGeometryCollision. It seams that the customer geometry cannot collide with the particle system but can with the box or convex shape. I just wonder if this is not supported or I need to do something else such as enable some flags or change somethings in the filter shader.

**Comment by xermao** [comment]
https://github.com/NVIDIA-Omniverse/PhysX/discussions/34
have you read this?
